astartes/main.py

```diff
@@ -149,16 +149,24 @@ def _extrapolative_sampling(
     # unlike interpolative, cannot calculate n_train_samples here
     # since it will vary based on cluster_lengths
 
-    # can't break up clusters
-    cluster_counter = sampler_instance.get_sorted_cluster_counter()
+    # largest clusters must go into largest set, but smaller ones can optionally
+    # be shuffled
+    cluster_counter = None
+    if sampler_instance.get_config("random_state") is not DEFAULT_RANDOM_STATE:
```

Comment:
Does this mean that if the random_state is 42 (our current value for `DEFAULT_RANDOM_STATE`) then the clusters will not actually be shuffled?